server/media_types.go

```diff
@@ -0,0 +1,6 @@
+package server
+
+const (
```

Comment:
i would make this file a bit more generic and name it `consts.go`

Reply:
Renamed to `constants.go`.